Makefile-test-ocp.mk

```diff
@@ -32,11 +34,35 @@ GINKGO = $(PROJECT_DIR)/bin/ginkgo
 ginkgo-ocp: ## Download ginkgo locally if necessary.
 	@GOBIN=$(PROJECT_DIR)/bin GO111MODULE=on $(GO_CMD) install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@$(GINKGO_OCP_VERSION)
 
+ENVTEST = $(PROJECT_DIR)/bin/setup-envtest
+.PHONY: envtest
+envtest-ocp: ## Download envtest-setup locally if necessary.
+	@GOBIN=$(PROJECT_DIR)/bin GO111MODULE=on $(GO_CMD) install sigs.k8s.io/controller-runtime/tools/setup-envtest@$(ENVTEST_OCP_VERSION)
+
+GOTESTSUM = $(PROJECT_DIR)/bin/gotestsum
+.PHONY: gotestsum
+gotestsum-ocp: ## Download gotestsum locally if necessary.
+	@GOBIN=$(PROJECT_DIR)/bin GO111MODULE=on $(GO_CMD) install gotest.tools/gotestsum@$(GOTESTSUM_OCP_VERSION)
+
 YQ = $(PROJECT_DIR)/bin/yq
 .PHONY: yq-ocp
 yq-ocp: ## Download yq locally if necessary.
 	@GOBIN=$(PROJECT_DIR)/bin GO111MODULE=on $(GO_CMD) install -mod=mod github.com/mikefarah/yq/v4@$(YQ_OCP_VERSION)
 
+.PHONY: test-ocp
+test-ocp: gotestsum-ocp ## Run tests.
+	$(GOTESTSUM) --junitfile $(ARTIFACTS)/junit.xml -- $(GOFLAGS) $(GO_TEST_FLAGS) $(shell $(GO_CMD) list ./... | grep -v '/test/') -coverpkg=./... -coverprofile $(ARTIFACTS)/cover.out
+
+.PHONY: test-integration-ocp
+test-integration: envtest-cop ginkgo-ocp dep-crds kueuect-ocp ginkgo-top-ocp ## Run integration tests for all singlecluster suites.
```

Comment:
Did you run this target locally? I suspect you might need a separate target for `dep-crds` unless you figured out the deps properly.

Reply:
updated. And ran locally and it worked